src/libprojectM/ProjectM.cpp

```diff
@@ -607,7 +564,7 @@ int ProjectM::initPresetTools(int gx, int gy)
     // Start the iterator
     if (!m_presetPos)
```

Comment:
Is init called more than once? If not, is there any reason for this check? If yes, maybe it shouldn't be called init...

Reply:
Unsure, probably was meant to in the past, don't know. Implementation cleanup is something we should certainly do later, now I'm just focusing on the header.

Reply:
Yes, certainly no need to check. Will remove it alongside other dead code in this method.